server/src/main/java/dev/slimevr/tracking/processor/HumanPoseManager.java

```diff
@@ -653,10 +665,78 @@ public void resetTrackersYaw(String resetSourceName) {
 				server
 					.getVMCHandler()
 					.alignVMCTracking(getRootNode().getWorldTransform().getRotation());
+				logTrackersDrift();
 			}
 		}
 	}
 
+	private void logTrackersDrift() {
+		if (timeAtLastReset == 0L)
+			timeAtLastReset = System.currentTimeMillis();
+
+		// Get time since last reset in seconds
+		long timeSinceLastReset = (System.currentTimeMillis() - timeAtLastReset) / 1000L;
+		timeAtLastReset = System.currentTimeMillis();
+
+		// Build String for trackers drifts
+		StringBuilder trackersDriftText = new StringBuilder();
+		for (Tracker tracker : server.getAllTrackers()) {
+			if (
+				tracker.getNeedsReset()
+					&& tracker.getResetsHandler().getLastResetQuaternion() != null
+			) {
+				// Get the difference between last reset and now
+				Quaternion difference = tracker
+					.getRotation()
+					.times(tracker.getResetsHandler().getLastResetQuaternion().inv());
+				// Get the pure yaw
+				float trackerDriftAngle = Math
+					.abs(
+						FastMath.atan2(difference.getY(), difference.getW())
+							* 2
+							* FastMath.RAD_TO_DEG
+					);
+				// Fix for polarity or something
+				if (trackerDriftAngle > 180)
+					trackerDriftAngle = Math.abs(trackerDriftAngle - 360);
+				// Round it to 4 decimal places
+				trackerDriftAngle = Math.round(trackerDriftAngle * 10000f) / 10000f;
+
+				// Calculate drift per minute
+				float driftPerMin = trackerDriftAngle / (timeSinceLastReset / 60f);
+				// Round it to 4 decimal places
+				driftPerMin = Math.round(driftPerMin * 10000f) / 10000f;
+
+				trackersDriftText.append(tracker.getName());
+				TrackerPosition trackerPosition = tracker.getTrackerPosition();
+				if (trackerPosition != null)
+					trackersDriftText.append(" (").append(trackerPosition.name()).append(")");
+
+				trackersDriftText
+					.append(", ")
+					.append(trackerDriftAngle)
+					.append(" deg (")
+					.append(driftPerMin)
+					.append(" deg/min) | ");
```

Comment:
The trailing `|` could be prevented by checking the StringBuilder for any content at the start of the loop, and only appending it if there is content.